src/status_im/subs/wallet/wallet.cljs

```diff
@@ -39,6 +39,12 @@
  :<- [:wallet/ui]
  :-> :scanned-address)
 
+(rf/reg-sub
+ :wallet/tokens
+ :<- [:wallet]
+ :-> :tokens)
+
+;; TODO(alwx): check how it's used
```

Comment:
Can we add more context of this TODO? Maybe add a GitHub issue to keep track of follow ups?